api/db-cdr/generate-cdr/libproject/devstart.rb

```diff
@@ -16,7 +16,7 @@
         :dest_cdr_project => "fc-aou-cdr-synth-test",
         :auth_domain_group_email => "GROUP_all-of-us-registered-test@dev.test.firecloud.org",
       },
-      "controlled_test" => {
+      "controlled" => {
```

Comment:
Do you also need to modify this line:
https://github.com/all-of-us/workbench/blob/master/api/db-cdr/generate-cdr/libproject/devstart.rb#L308

Reply:
I will check, thanks.

Reply:
yes, updated